content/v2.0-preview/get-started/get-started-with-managed-resources.md

```diff
@@ -188,44 +189,42 @@ spec:
 EOF
 ```
 
-The {{< hover label="xr" line="2">}}apiVersion{{< /hover >}} and 
-{{< hover label="xr" line="3">}}kind{{</hover >}} are from the provider's CRDs.
-
-
-The {{< hover label="xr" line="5">}}metadata.generateName{{< /hover >}} value is the 
-name of the created S3 bucket in AWS.  
-This example uses the generated name `crossplane-bucket-<hash>` in the 
-{{< hover label="xr" line="5">}}$bucket{{</hover >}} variable.
+The {{< hover label="xr" line="6">}}metadata.generateName{{< /hover >}} gives a
+pattern that the provider will use to create a unique name for the bucket in S3.
+The generated name will look like `crossplane-bucket-<hash>`.
 
-The {{< hover label="xr" line="8">}}spec.forProvider.region{{< /hover >}} tells 
-AWS which AWS region to use when deploying resources.  
-
-The region can be any 
-[AWS Regional endpoint](https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints) code.
-
-Use `kubectl get buckets` to verify Crossplane created the bucket.
+Use `kubectl -n crossplane-aws-app get buckets.s3.aws.m.upbound.io` to verify Crossplane created the bucket.
 
 {{< hint type="tip" >}}
-Crossplane created the bucket when the values `READY` and `SYNCED` are `True`.  
-This may take up to 5 minutes.  
+Crossplane created the bucket when the values `READY` and `SYNCED` are `True`.
+This may take up to 5 minutes.
 {{< /hint >}}
 
 ```shell {copy-lines="1"}
-kubectl get buckets
-NAME                      READY   SYNCED   EXTERNAL-NAME             AGE
-crossplane-bucket-hhdzh   True    True     crossplane-bucket-hhdzh   5s
+kubectl -n crossplane-aws-app get buckets.s3.aws.m.upbound.io
+NAME                      SYNCED   READY   EXTERNAL-NAME             AGE
+crossplane-bucket-7tfcj   True     True    crossplane-bucket-7tfcj   3m4s
 ```
 
 ## Delete the managed resource
-Before shutting down your Kubernetes cluster, delete the S3 bucket just created.
+Before shutting down your Kubernetes cluster, delete the S3 bucket that was just created.
```

Comment:
```suggestion
Delete the S3 bucket that was just created.
```

Nit: Leading with "Before shutting down your Kubernetes cluster" feels weird to me for some reason here. What if they're using an existing cluster and not e.g. `kind`?

I'd suggest leaving it out. Maybe we could add a `{{< hint >}}` mentioning that it's important to delete the bucket before uninstalling Crossplane/deleting cluster/etc.

Reply:
I'll remove this language from the mainline path and include it in a `<hint>` box.